### ISSUE TITLE
[feature request] add lastfm scrabble support like this app https://github.com/KRTirtho/spotube

### ISSUE THREAD

**Issue by rajarshikhatua** [Malopieds/InnerTune#439]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

last fm scrabble with last fm login in the app

### Why do you want this feature?

i use last fm to track my music

**Comment by unsignedchar-256** [comment]
https://github.com/Malopieds/InnerTune?tab=readme-ov-file#q-how-to-scrobble-music-to-lastfm-librefm-listenbrainz-or-gnu-fm

**Comment by rajarshikhatua** [comment]
> https://github.com/Malopieds/InnerTune?tab=readme-ov-file#q-how-to-scrobble-music-to-lastfm-librefm-listenbrainz-or-gnu-fm

my point was without installing additional apps

**Comment by unsignedchar-256** [comment]
> > https://github.com/Malopieds/InnerTune?tab=readme-ov-file#q-how-to-scrobble-music-to-lastfm-librefm-listenbrainz-or-gnu-fm
> 
> my point was without installing additional apps

I guess [Malopieds](https://github.com/Malopieds/InnerTune) and [Zion Huang](https://z-huang/InnerTune)'s InnerTune already said that in the `README.md` file because they may not want or have an idea to add LastFM scrobble feature.

**Comment by zettaexa** [comment]
Pano Scrobbler

**Comment by Malopieds** [comment]
Yes won't be included in the app, you can use as specified, Piano Scrobbler
